make/utils.cmake

```diff
@@ -163,6 +163,48 @@ function (place_shared_lib_in_lib_dir target)
     ARCHIVE_OUTPUT_DIRECTORY${location_suffix} "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 endfunction ()
 
+function (check_avx_processor_and_compiler_support out)
+  if (WIN32)
+    message(FATAL_ERROR "Windows not supported yet.")
+  endif ()
+  include(CheckCSourceRuns)
+  # XXX i#1312: add Windows support.
```

Comment:
Comment should go up by the `if (WIN32)`.  Ditto below.

Reply:
Done.